### PR TITLE
Added CORS policy for tts get request

### PR DESCRIPTION
**Pull request by insualk**
Issue: #633

### CODE REVIEW

File: server/backend/Startup.cs [hunk 1]
```diff
@@ -26,6 +26,10 @@ public void ConfigureServices(IServiceCollection services)
                 builder.AllowAnyOrigin().AllowAnyMethod().AllowAnyHeader().WithExposedHeaders(
                     "Grpc-Status", "Grpc-Message", "Grpc-Encoding", "Grpc-Accept-Encoding");
             }));
+            services.AddCors(o => o.AddPolicy("AllowAudioFileGet", builder =>
```
Comment (ztl8702) [inline server/backend/Startup.cs:29]: merge with the previous call to `services.AddCors(`? 

i.e. 
```
services.AddCors(o => {
  o.AddPolicy...
  o.AddPolicy...
```

Comment (ztl8702) [reply server/backend/Startup.cs:29]: Or, I think you can just reuse the `AllowAll` policy`

**Comment by ztl8702** [comment]
> p.s. should we exclude server/backend/.vscode?

No. I think we should check `.vscode/` into git. I think that's what most projects are doing, including [VSCode itself](  https://github.com/microsoft/vscode/blob/master/.vscode/launch.json).

**Comment by ztl8702** [comment]
Nice work! Free free to merge.

**Comment by ztl8702** [comment]
tip: 

We have mergify bot configured to automerge a PR when tests pass.

You can write the commit message in a subheading called `Commit message` in the PR description ([example](https://github.com/MindongLab/yngdieng/pull/628#issue-486178434) ), and tag the PR with `automerge`. Mergify will handle the rest for you.

**Comment by insualk** [comment]
> tip:
> 
> We have mergify bot configured to automerge a PR when tests pass.
> 
> You can write the commit message in a subheading called `Commit message` in the PR description ([example](https://github.com/MindongLab/yngdieng/pull/628#issue-486178434) ), and tag the PR with `automerge`. Mergify will handle the rest for you.

What's the purpose of adding the commit message subheading?

**Comment by ztl8702** [comment]
> What's the purpose of adding the commit message subheading?

The default commit message generated by github is not always what you'd expect. Since we ask mergify to merge automatically, we want to tell mergify exactly what commit message to use, rather than using the github default.

According to mergify docs: "you can override the default commit message... add a section in the pull request body that starts with Commit Message." 

https://doc.mergify.io/actions.html#commit-message

**Comment by ztl8702** [comment]
I just switched the automerge bot from mergify to [kodiak](https://kodiakhq.com/), which doesn't require you to add a "Commit Message" subheading anymore. @insualk

**Pull request merged**